src/opbox.rs

```diff
@@ -81,6 +83,26 @@ pub enum OpBox {
         /// Config param for decomposition of Pauli exponentials.
         cx_config: String,
     },
+    /// An unordered collection of Pauli exponentials that can be synthesised in
+    /// any order, causing a change in the unitary operation. Synthesis order
+    /// depends on the synthesis strategy chosen only.
+    TermSequenceBox {
+        id: BoxID,
+        /// List of Symengine expressions.
+        pauli_gadgets: Vec<(Vec<String>, String)>,
+        /// Synthesis strategy. See [`PauliSynthStrat`].
+        #[serde(default)]
+        synthesis_strategy: PauliSynthStrat,
```

Comment:
Can we make the names match those used in TKET serialization ( https://github.com/CQCL/tket/blob/a2f6fab8a57da8787dfae94764b7c3a8e5779024/schemas/circuit_v1.json#L832 )?

Reply:
I did a full pass on the schema to ensure our definition was OK.

Apparently some of the already-present box definitions where missing or incorrect, so I also fixed them.

Reply:
Unfortunately #36 won't be able to catch mismatching attributes.

To validate that we'd need to add tests that construct each possible `pytket` optype and check it against the rust code :thinking:

I may add a low-priority issue for that.